packages/muelu/adapters/xpetra/MueLu_RefMaxwell_def.hpp

```diff
@@ -786,6 +787,11 @@ namespace MueLu {
         if (!reuse) {
           ParameterList& userParamList = precList22_.sublist("user data");
           userParamList.set<RCP<RealValuedMultiVector> >("Coordinates", Coords_);
+          // If we detected no boundary conditions, the (2,2) problem is singular
+          if (BCrowcount_ == 0 &&
+              !precList22_.isSublist("coarse: params") &&
```

Comment:
Don't you want this sublist to exist?

Reply:
The logic is: if we didn't already set the parameter before, then set it here. I don't want to clobber the preexisting value.

Reply:
:vulcan_salute: